src/modules/tasks.js

```diff
@@ -0,0 +1,105 @@
+import resetIndex from './index-resetter.js';
+import { loadTasks, saveTask } from './Save-and-load.js';
+
+export default class Task {
+  constructor(id, description, completed) {
+    this.index = id;
+    this.description = description;
+    this.completed = completed;
+    return { id: this.index, description: this.description, completed: this.completed };
+  }
+}
+
+export const addTask = (element, tasks) => {
+  tasks.push(element);
+  // save books to local storage
+  saveTask(tasks, 'todo-list');
+};
+
+export const removeTask = (id, tasks) => {
```

Comment:
Delete a task functionality is still not working although I can remove tasks which are checked by clicking `Clear All Completed`. Kindly make sure this requiremet is met before asking for another review.

Reply:
dear @SajjadAhmad14 , @OLIPLICHE  kindly what this video and tell me if anything doesn't work as required. let's notice that I'm using the same code; no new commits